apps/astarte_realm_management_api/priv/static/astarte_realm_management_api.yaml

```
@@ -464,6 +464,30 @@ paths:
           $ref: '#/components/responses/TriggerDeliveryPolicyCurrentlyUsedError'
         '500':
           description: Internal Server Error.
+  '/{realm_name}/devices/{device_id}':
```

We can add parameters
```suggestion
  '/{realm_name}/devices/{device_id}':
    parameters:
      - $ref: '#/components/parameters/Realm'
      - $ref: '#/components/parameters/DeviceID'
```
And into `parameters` below
```
    DeviceID:
      name: device_id
      in: path
      description: Target realm
      required: true
      schema:
        type: string
        - name: device_id
          in: path
          description: Device ID
          required: true
          schema:
            type: string
```